modules/custom/d_demo/pages/product.yml

```diff
@@ -51,7 +63,7 @@ field_page_section:
           data: <p>Experience marked by 20 years on the market<br />- wide range of products</p>
```

Comment:
Those fields here should than be deleted since you drop them in features from paragraph d_p_side_by_side :)

Reply:
done :)